lib/db.js

```diff
@@ -424,23 +444,46 @@ Db.prototype.collection = function(name, options, callback) {
  * @param {array} [options.pipeline] An array that consists of the aggregation pipeline stage. create creates the view by applying the specified pipeline to the viewOn collection or view on MongoDB 3.4 or higher.
  * @param {object} [options.collation] Specify collation (MongoDB 3.4 or higher) settings for update operation (see 3.4 documentation for available fields).
  * @param {ClientSession} [options.session] optional session to use for this operation
+ * @param {object} [options.promiseLibrary] A Promise library class the application wishes to use such as Bluebird, must be ES6 compatible
```

Comment:
let's make a ticket to only allow users to pass in a promise library to `MongoClient`, and stop documenting it on individual methods.

Reply:
`promiseLibrary` is only documented on this operation because of [this code](https://github.com/mongodb/node-mongodb-native/pull/1864/files/07a06ae484d05036fa5be891791fc8893e3fb359#diff-be228df81b99a384aa50ad50bde7fca0L439).

Reply:
yeah I just don't think its very useful, just wanted a ticket to track getting rid of it everywhere

Reply:
I made NODE-1753 to track this.